stubs/sso/__init__.py

```diff
@@ -0,0 +1,150 @@
+import base64
```

Comment:
We probably want a decent comment at the top of this file, or a README for the sso stub directory, explaining what this is for and why